MoviesTVSentiments/app/src/androidTest/java/com/google/moviestvsentiments/usecase/assetList/AssetListFragmentTest.java

```diff
@@ -0,0 +1,146 @@
+package com.google.moviestvsentiments.usecase.assetList;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static com.google.moviestvsentiments.assertions.RecyclerViewItemCountAssertion.withItemCount;
+
+import android.content.Intent;
+import android.os.Bundle;
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule;
+import androidx.test.core.app.ApplicationProvider;
+import com.google.moviestvsentiments.HiltTestActivity;
+import com.google.moviestvsentiments.R;
+import com.google.moviestvsentiments.di.DatabaseModule;
+import com.google.moviestvsentiments.model.AssetType;
+import com.google.moviestvsentiments.model.SentimentType;
+import com.google.moviestvsentiments.service.database.SentimentsDatabase;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import javax.inject.Inject;
+import dagger.hilt.android.testing.HiltAndroidRule;
+import dagger.hilt.android.testing.HiltAndroidTest;
+import dagger.hilt.android.testing.UninstallModules;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import com.google.moviestvsentiments.HiltFragmentScenario;
+import com.google.moviestvsentiments.usecase.signin.SigninActivity;
+import com.google.moviestvsentiments.util.AssetUtil;
+
+@UninstallModules(DatabaseModule.class)
+@HiltAndroidTest
+@RunWith(JUnitParamsRunner.class)
+public class AssetListFragmentTest {
+
+    private static Bundle createFragmentArgs(SentimentType sentimentType) {
+        Bundle arguments = new Bundle();
+        arguments.putSerializable("sentimentType", sentimentType);
+        return arguments;
+    }
+
+    @Rule
+    public HiltAndroidRule hiltAndroidRule = new HiltAndroidRule(this);
+
+    @Rule
+    public InstantTaskExecutorRule instantTaskExecutorRule = new InstantTaskExecutorRule();
+
+    @Inject
+    SentimentsDatabase database;
+
+    @Before
+    public void setUp() {
+        hiltAndroidRule.inject();
+    }
+
+    @Test
+    @Parameters(method = "startsWithEmptyListValues")
+    public void assetListFragment_startsWithEmptyList(SentimentType sentimentType, int listId) {
+        HiltFragmentScenario.launchHiltFragment(AssetListFragment.class,
+                createFragmentArgs(sentimentType));
+
+        onView(withId(listId)).check(withItemCount(0));
+    }
+
+    private Object[] startsWithEmptyListValues() {
+        return new Object[] {
+            new Object[] {SentimentType.UNSPECIFIED, R.id.movies_list},
+            new Object[] {SentimentType.THUMBS_UP, R.id.movies_list},
+            new Object[] {SentimentType.THUMBS_DOWN, R.id.movies_list},
+            new Object[] {SentimentType.UNSPECIFIED, R.id.tvshows_list},
+            new Object[] {SentimentType.THUMBS_UP, R.id.tvshows_list},
+            new Object[] {SentimentType.THUMBS_DOWN, R.id.tvshows_list}
```

Comment:
What about just using sentiment as the param, and check both list in the test?

Reply:
Also can you put all params before their test? That can help readability. Thanks